.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
Please remove this file and amend the commit.

Reply:
Alright. Done.